plugins/modules/keycloak_client_rolemapping.py

```diff
@@ -320,25 +320,25 @@ def main():
         if state == 'present':
             # Assign roles
             result['changed'] = True
-            if module._diff:
-                result['diff'] = dict(before=assigned_roles_before, after=update_roles)
```

Comment:
Removing this will break diff mode when combined with check mode. (Same below.)

Reply:
Uh, you're right. I'll need to pre-calculate the result in that case.